bridgetown-website/bin/netlify.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+# Taken from https://docs.netlify.com/configure-builds/file-based-configuration/#inject-environment-variable-values
+
+echo "Updating netlify.toml with references to our built files"
+
+CSS_PATH=`find output/_bridgetown/static/css/*.css -type f | sed -e 's,output\/,/,g'`
+JS_PATH=`find output/_bridgetown/static/js/*.js -type f | sed -e 's,output\/,/,g'`
+
+echo "CSS Path: ${CSS_PATH}"
+echo "JS Path: ${JS_PATH}"
+
+sed -i s,CSS_PATH,${CSS_PATH},g netlify.toml
```

Comment:
What this'll do is update our `netflify.toml` after build with the CSS/JS paths. So we can tell the browser in the HTTP header which files to expect.